README.md

```diff
@@ -18,7 +28,7 @@ This release version of the library has made a few changes to standardize its us
 ## Documentation
 
 * See the tango documentation [here](http://tango-controls.readthedocs.io/en/latest/administration/services/hdbpp/index.html#hdb-an-archiving-historian-service) for broader information about the HB++ archiving system and its integration into Tango Controls
-* Libhdbpp-Cassandra [CHANGELOG.md](https://github.com/tango-controls-hdbpp/libhdbpp/blob/master/CHANGELOG.md) contains the latest changes both released and in development.
```

Comment:
I guess this link was removed because it was always pointing to master, even when working on a branch...
There are other links like that in this file. (INSTALL.md for instance).
I think links are nice... They should probably be added again before merging to master.

Reply:
Maybe we could try to use relative links and see whether it works as expected:
https://github.com/blog/1395-relative-links-in-markup-files

Reply:
I can try this. Its starting to expand the patch and go outside its focus a little.